azure-pipelines.yml

```diff
@@ -203,33 +157,26 @@ jobs:
         condition: always()
 
   - job: Screener
-    dependsOn: Build
-    # keeping this at microsoft hosted because this doesn't require builds
-    pool:
-      vmImage: 'Ubuntu 18.04'
+    workspace:
+      clean: all
     steps:
-      # No need for checkout since we're using build artifacts.
-      - checkout: none
-
-      # - template: azure-pipelines.cache.yml
-
-      - template: azure-pipelines.artifacts.yml
-        parameters:
-          artifact: Build-PR-$(Build.BuildNumber)
-          buildId: $(Build.BuildId)
+      - script: |
+          yarn
+        displayName: yarn
 
       - template: azure-pipelines.tools.yml
 
+      - template: .devops/templates/pr-target-branch.yml
+
       - script: |
           git config --global user.email "fabrictactical@microsoft.com"
           git config --global user.name "Fabric Tactical"
-          yarn vrtest -- --debug
+        displayName: config git user and email
+
+      - script: |
+          yarn lage vrtest --debug --since $(target_branch) --no-deps --verbose
```

Comment:
This gets us building & getting to vrtest as quickly as possible. Not all things need to be built, so it's scoped to that (naturally as well because only one project has vrtest task I think)

Reply:
Okay, so this is basically like our current `yarn buildto`? (I couldn't immediately tell based on experience with other tools what this would do.)

Reply:
yes, that's right.